bot/bot.py

```diff
@@ -10,6 +12,13 @@
 class SirRobin(commands.Bot):
     """Sir-Robin core."""
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.name = "Sir-Robin"
+        self._guild_available = asyncio.Event()
+        self.loop.create_task(self.check_channels())
+        self.loop.create_task(self.send_log(self.name, "Connected!"))
```

Comment:
Should use [`botcore.utils.scheduling.create_task()`](https://bot-core.pythondiscord.com/output/botcore.utils.scheduling.html#botcore.utils.scheduling.create_task) for both of these, so that exceptions are raised when hit, rather that when the loop closes.

Reply:
Addressed in [b799837](https://github.com/python-discord/sir-robin/pull/16/commits/b799837c43e953aa45909a140d5f4b23790e3369)